packages/generator-electrode/component/index.js

```diff
@@ -181,6 +182,15 @@ var ReactComponentGenerator = yeoman.Base.extend({
       this.template("packages/component/_readme.md", this.rootPath + "README.md");
     },
     component: function() {
+      let getDemoFilePath = function(filepath) {
+        try {
```

Comment:
@jchip  If the filepath does not exists, it will throw the error similar to below.

The try catch is a double check for the files copied form demo-helper. We have the same function in app generator and I think it is needed for component as well, please see here: https://github.com/electrode-io/electrode/blob/a2e995745825a29a3280d5e26ee597a7845aa4e2/packages/generator-electrode/generators/app/index.js#L250-L257
 
And here is the error trying to catch (if the file does not exists)

```bash
events.js:160
      throw er; // Unhandled 'error' event
      ^
AssertionError: Trying to copy from a source that does not exist: /Users/s0d00px/electrode/packages/generator-electrode/node_modules/electrode-demo-helper/test
    at EditionInterface.exports._copySingle (/Users/s0d00px/electrode/packages/generator-electrode/node_modules/yeoman-generator/node_modules/mem-fs-editor/lib/actions/copy.js:45:3)
    at EditionInterface.exports.copy (/Users/s0d00px/electrode/packages/generator-electrode/node_modules/yeoman-generator/node_modules/mem-fs-editor/lib/actions/copy.js:23:17)
    at EditionInterface.module.exports [as copyTpl] (/Users/s0d00px/electrode/packages/generator-electrode/node_modules/yeoman-generator/node_modules/mem-fs-editor/lib/actions/copy-tpl.js:10:8)
    at constructor.component (/Users/s0d00px/electrode/packages/generator-electrode/component/index.js:210:15)
    at Object.<anonymous> (/Users/s0d00px/electrode/packages/generator-electrode/node_modules/yeoman-generator/lib/base.js:431:23)
    at /Users/s0d00px/electrode/packages/generator-electrode/node_modules/yeoman-generator/node_modules/run-async/index.js:25:25
    at /Users/s0d00px/electrode/packages/generator-electrode/node_modules/yeoman-generator/node_modules/run-async/index.js:24:19
    at /Users/s0d00px/electrode/packages/generator-electrode/node_modules/yeoman-generator/lib/base.js:432:9
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
```